app/serializers/institution_search_result_serializer.rb

```diff
@@ -29,10 +29,18 @@ class InstitutionSearchResultSerializer < ActiveModel::Serializer
   attribute :distance
   attribute :accredited
   attribute :vet_tec_provider
+  attribute :programs
```

Comment:
do you need the entire list of them or just a count?

Reply:
We need the whole list to get `lengthInHours`